src/server.h

```diff
@@ -470,6 +474,12 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 	void DeleteClient(u16 peer_id, ClientDeletionReason reason);
 	void UpdateCrafting(Player *player);
 
+	// This returns the answer to the sender of wmessage, or "" if there is none
+	std::wstring handleChat(const std::string &name, const std::wstring &wname,
+		const std::wstring &wmessage,
+		u16 peer_id_to_avoid_sending = PEER_ID_INEXISTENT);
```

Comment:
Uhh... what possible purpose could there be for that?
Code should be as standard and non-weird as possible.